rpcs3/Crypto/utils.cpp

```diff
@@ -2,8 +2,9 @@
 // Licensed under the terms of the GNU GPL, version 3
 // http://www.gnu.org/licenses/gpl-3.0.txt
 
-#include "utils.h"
-#include <stdio.h>
+#include "utils.h"
+#include <cstring>
+#include <stdio.h>
```

Comment:
Not sure. That line was already there (I'm not sure why that appeared as a changed line?).